GamingTheSyllabus/Assets/Scripts/MainMenuManager.cs

```diff
@@ -24,7 +24,8 @@ public static void StartGame()
         DatabaseManager.CurrentUserEmail = email;
         DatabaseManager.CurrentSyllabusCode = syllabusCode;
 
-        DatabaseManager.AttemptToDownloadSyllabusInformation(ContinueToGame, ShowDownloadFailure);
+        DatabaseManager.VerifyAndWriteToDatabase(ContinueToGame, ShowDownloadFailure, false);
+        // DatabaseManager.AttemptToDownloadSyllabusInformation(ContinueToGame, ShowDownloadFailure);
```

Comment:
I had added in a syllabus info file that gets downloaded to check if the folder actually exists in cloud storage as a way of verifying the syllabus code. I figured since I need it anyway for verification we could include some data that may be used at some point (prof name, class, semester, etc) but tbh unless we decide to use it im fine with leaving this commented out